packages/terra-clinical-onset-picker/package.json

```diff
@@ -37,7 +37,7 @@
     "terra-date-picker": "^3.0.0",
     "terra-doc-template": "^2.0.0",
     "terra-form-field": "^3.0.0",
-    "terra-form-fieldset": "^2.0.0",
+    "terra-form-fieldset": "^2.12.0",
```

Comment:
lets keep this at 2.0.0. These were recently changed to the latest mvb number to avoid mismatching peer depenendy warnings. Ref: https://github.com/cerner/terra-clinical/pull/448

Reply:
Changed this by thinking that we might need to update the version of component to latest version to get latest code changes add to component. but I am able to see new prop added to form-fieldset in version 2.0.0 . Reverted the version of terra-form-fieldset to 2.0.0 : 3056a799ecd754f09a91f7ffd95534200fbf5a6f